### PR TITLE
Fixed-size-array padding fix

### LINKED ISSUE #56: Arrays as params are not supported
Sample contract:

```rust
#[eth_abi(TokenEndpoint, TokenClient)]
pub trait SampleContractInterface {
  fn constructor(&mut self);
  fn createRequest(&mut self, serviceNumber: [u8; 30], date: u64);
}

pub struct SampleContract;

impl SampleContractInterface for SampleContract {
	fn constructor(&mut self) {}

	fn createRequest(&mut self, serviceNumber: [u8; 30], date: u64) {}
}
```

All tests are passing, but when it's called via `web3` api it returns an error.

Logs don't provide a lot of information, however I post them too

```json
{
  "transactionHash": "0xeb0885071aa00c8ec9c6df9620af67c22aa0012daef881f607bc5c8a4df8534c",
  "transactionIndex": "0x0",
  "blockHash": "0xe5f5d1018bbcf228695c8cfcd7f548f434384df6a4e0ee5bfba44b488d93d22e",
  "blockNumber": "0x5d47",
  "cumulativeGasUsed": "0x4630c0",
  "gasUsed": "0x4630c0",
  "contractAddress": null,
  "status": "0x0",
  "logs": []
}
```

I expect it to be working via web3 calls.

### PR DESCRIPTION
**Pull request by fckt**
Fixed-size array elements should have left-aligned padding.

From the spec:

```
bytes, of length k (which is assumed to be of type uint256):

enc(X) = enc(k) pad_right(X), i.e. the number of bytes is encoded as a uint256 followed by the actual value of X as a byte sequence, followed by the minimum number of zero-bytes such that len(enc(X)) is a multiple of 32.
```

```
bytes<M>: enc(X) is the sequence of bytes in X padded with zero-bytes to a length of 32.
```

Also `ethereumjs-abi` implementation proves that there is no right-padding for fixed-size arrays:
https://github.com/ethereumjs/ethereumjs-abi/blob/master/lib/index.js#L135

In contrast with a non-fixed-size arrays:
https://github.com/ethereumjs/ethereumjs-abi/blob/master/lib/index.js#L142

fixes #56

### CODE REVIEW

File: src/eth/tests.rs [hunk 1]
```diff
@@ -113,6 +118,15 @@ fn sample1_decode() {
 	assert_eq!(v2, true);
 }
 
+#[test]
+fn fixed_size_decode() {
+	let payload: &[u8] = &[
+		0x45, 0x01, 0x03, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x08, 0x00, 0x00,
+	];
+	let array = single_decode::<[u8; 30]>(payload);
+	assert_eq!(array, [0x45, 0x01, 0x03, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x08]);
```
Comment (NikVolf) [inline src/eth/tests.rs]: better use 30 non-zero values, not 3

**Comment by NikVolf** [comment]
test fails

File: src/eth/tests.rs (hunk 1 above)
Comment (fckt) [reply src/eth/tests.rs]: the last byte is non-zero

Comment (NikVolf) [reply src/eth/tests.rs]: cool, maybe make it two lines so everyone would see

**Pull request merged**